blog/en-us/Apache_dolphinScheduler_2.0.3.md

```diff
@@ -0,0 +1,146 @@
+---
+title:Apache DolphinScheduler 2.0.3 Release Announcement: DingTalk alert plugin adds signature verification, and supports data sources to obtain links from multiple sessions
+keywords: Apache,DolphinScheduler,scheduler,big data,ETL,airflow,hadoop,orchestration,dataops,2.0.3
+description:Today, Apache DolphinScheduler announced the official release of version 2.0.3. In this version,
+---
+# Apache DolphinScheduler 2.0.3 Release Announcement: DingTalk alert plugin adds signature verification, and supports data sources to obtain links from multiple sessions
+
+<div align=center>
+<img src="/img/2.0.3/2022-1-2701/1.png"/>
+</div>
+
+> Today, Apache DolphinScheduler announced the official release of version 2.0.3. In this version, DingTalk alert plugin adds signature verification and enables data sources to get links from multiple sessions. In addition, 2.0.3 also optimizes cache management, complement time, data source password display in logs, etc., and fixes several key vulnerabilities.
```

Comment:
`vulnerabilities` Usually only for security issues.

Reply:
ok